src/components/navlink.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+
+type Props = {
+  className?: string;
+  href: string;
+  children: string;
+};
+
+const NavLink: React.FunctionComponent<Props> = ({
+  className,
```

Comment:
Lint check is saying that this className is never used. Can be removed or be used in className of Link tag on line 20